app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="color_primary">#3F51B5</color>
-    <color name="color_primaryDark">#303F9F</color>
-    <color name="color_accent">#FF4081</color>
+    <color name="color_primary">#0797dc</color>
+    <color name="color_primaryDark">#0797dc</color>
+    <color name="color_accent">#ffffff</color>
+    <color name="color_blue_400">#0797dc</color>
+    <color name="color_black">#000000</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_black` appears to be unused
 Unused resources make applications larger and slow down builds.